### PR TITLE
[DATABASE CHANGE] beta deploy

### PR DESCRIPTION
**Pull request by kijk2869**
```sql
ALTER TABLE users ADD aflco VARCHAR(3);
ALTER TABLE users ADD major LONGTEXT;
```

### CODE REVIEW

**Comment by wonderlandpark** [comment]
> ```sql
> ALTER TABLE users ADD aflco VARCHAR(3);
> ALTER TABLE users ADD major LONGTEXT;
> ```

No Default Value?

**Comment by kijk2869** [comment]
> > ```sql
> > ALTER TABLE users ADD aflco VARCHAR(3);
> > ALTER TABLE users ADD major LONGTEXT;
> > ```
> 
> No Default Value?

Default value is null, and all data is managed by tortoise-orm

File: bot.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 
 class Bot(commands.Bot):
-    __version__ = "1.1.1"
+    __version__ = "1.1.2"
```
Comment (SaidBySolo) [inline bot.py]: 1.2.0이 맞는듯합니다

고등학교지원 +

**Review by SaidBySolo — APPROVED** [review]

File: bot.py (hunk 1 above)
Comment (SaidBySolo) [reply bot.py]: 아 그리고 비공개 여부까지 패치 됬으면 1.3.0 ㄱ

**Comment by kijk2869** [comment]
@SaidBySolo have to alter table before merge

**Comment by SaidBySolo** [comment]
DROP TABLE

**Pull request merged**